prismic-configuration.js

```diff
@@ -1,8 +1,16 @@
+
 module.exports = {
+  
   // The repo name for your Prismic repository (found in the url of your repo: https://your-repo-name.prismic.io/)
   prismicRepo: 'your-repo-name',
 
+  // Release ID to build (requires access)
+  // releaseID: '',
```

Comment:
Will this work if you uncommented it? That way if someone adds it, they won't need to uncomment it.

Same for the accessToken below.

Reply:
When you work do you mean will it try to make a release build, nope
an empty string wouldn't match this if condition so it shouldn't try and fetch release documents.
https://github.com/angeloashmore/gatsby-source-prismic/blob/master/src/api.ts#L72

apiToken is passed to to prismic-javascript ... if prismic-javascript is okay with having empty tokens then I guess that's fine.